### PR TITLE
Renable HCRLateAttachWorkload tests on Windows

### PR DESCRIPTION
**Pull request by lumpfish**
Signed-off-by: Simon Rushton <srushton@redhat.com>

### CODE REVIEW

**Comment by lumpfish** [comment]
Fixes https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/236

**Comment by lumpfish** [comment]
Test runs of the HCRLateAttachWorkload tests with this PR:
jdk8
openj9
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4584/
hotspot
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4585/


jdk11
openj9
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4586/
hotspot
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4587/


jdk15
openj9
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4588/
hotspot
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4589/

File: system/otherLoadTest/playlist.xml [hunk 1]
```diff
@@ -217,7 +217,6 @@
 		<groups>
 			<group>system</group>
 		</groups>
-		<platformRequirements>^os.win</platformRequirements>
```
Comment (Mesbah-Alam) [inline system/otherLoadTest/playlist.xml:220]: Could you please remove the comment associated with the exclude too? ("")

**Review by Mesbah-Alam — COMMENTED** [review]
Please remove the comment

File: system/otherLoadTest/playlist.xml (hunk 1 above)
Comment (lumpfish) [reply system/otherLoadTest/playlist.xml:220]: Removed the comment

Comment (Mesbah-Alam) [reply system/otherLoadTest/playlist.xml:220]: Thanks!

**Review by Mesbah-Alam — APPROVED** [review]
LGTM.

**Pull request merged**